packages/ckeditor5-utils/src/locale.ts

```diff
@@ -165,7 +178,11 @@ export default class Locale {
  * @param values A value or an array of values that will fill message placeholders.
  * For messages supporting plural forms the first value will determine the plural form.
  */
-export type LocaleTranslate = ( message: string | Message, values?: number | string | ReadonlyArray<number | string> ) => string;
+export type LocaleTranslate = (
+	message: string | Message,
+	values?: number | string | ReadonlyArray<number | string>,
+	translations?: { [key: string]: string } | null
```

Comment:
Why 3rd parameter? Seems unused. The function of this type is initialized in this file in line 130. It has 2 parameters only.

Reply:
It is used right here: https://github.com/ckeditor/ckeditor5/pull/15734/files#diff-bee38257de0d7815896e6ced52d1dca3534beb0e4f4faa84182cdf4dbdaf7ce6R127-R129